src/FirebaseProject.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Laravel\Firebase;
+
+use Kreait\Firebase;
+
+class FirebaseProject
+{
+    /** @var \Kreait\Firebase\Factory $factory */
+    protected $factory;
+
+    /** @var array $config */
+    protected $config;
+
+    /** @var \Kreait\Firebase\Auth|null $auth */
+    protected $auth = null;
```

Comment:
```suggestion
    protected $auth;
```

`null` is already the default value - it of course doesn't matter in the functionality, but while I usually like being explicit, class properties that are null by default are one exception 🙈 (even if you'll certainly find occurrences where I don't follow my own rules 😂)

Reply:
Forgot to remove it when I removed the property types :)